src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -3449,7 +3458,7 @@ namespace <#= fullNamespace #>
         
         if (!String.IsNullOrEmpty(path))
         {
-            using (StreamWriter writer = new StreamWriter(path, true))
+            using (StreamWriter writer = new StreamWriter(path, false))

```

Comment:
>false [](start = 64, length = 5)

why change "true" to "false"?

Reply:
One of the bugs reported in the same issue is that the code generator was appending the string in the temp metadata file. This made the file too big if the code is generated several times. To fix this I changed the parameter in the constructor to *false* to overwrite the file instead of appending.